src/i18n/index.js

```diff
@@ -5,9 +5,11 @@ import VueI18n from 'vue-i18n';
 // then import the file and add it to the 'translations' const as seen below
 
 import en from './en';
+import uk from './uk'
```

Comment:
Isn't the official ISO shortcode for Ukraine `UA`? .. UK might get confused with United Kingdom

Reply:
I believe that's the shortcode for the country, not necessarily the language. The ISO 639-1 table found on Wikipedia shows that the Ukrainian language ID is `uk`.

`uk` in reference to United Kingdom comes into play with the `-` in the language ID: `en-US`, `en-UK` to denote the dialect I believe.

Reply:
@Mordil argh didn't see the ISO was part of the README .. I have always been using `ISO-3166` which specifies Ukraine as UA :)